### DISCUSSION TITLE
S1 discount before new season

### DISCUSSION THREAD

**Discussion by tbcxd** [chromiecraft/chromiecraft#6667] [Suggestions]
Hello

I would like to suggest to heavily reduce the prices of s1 gear before season 2 release as well as allowing s1 gear to be bought with honor once s2 is live.

Reasons to let people buy s1 at discounted prices:
1) It will allow everyone to be equally geared and have a fair season start.
2) There will be a short-term spike in arena activity as most people will try to take advantage of the discounted prices and get their 10 games done on all their toons.
3) everyone loves discounts, myself included :D

Reasons to let people buy s1 gear for honor once season2 is live:
1) Battlegrounds activity will increase, this is a no-brainer.
2) New players who want to be start playing on CC as well as fresh 70s or players with alts will not feel discouraged to join the pvp scene if they know they can get "arena ready" pretty quickly from 2 or 3 CTA weekends, as well as being able to already save their arena points from day1 in order to upgrade their s1 gear to s2.

Enabling these features should wipe all Arena Points/Honor and Marks from players to ensure a fresh and fair new season, if all the above don't get wiped, people who saved a bunch of honor/marks or arena points will have a big advantage over others.

The discounted prices on s1 gear as well as its honor price is subject to discussion in the comments under this post, so feel free to comment what you think the costs should be!

Thank you for reading,
Tryhard/Muscle.

**Comment by sassouiabd** [comment]
⛔ for previous arena gear only with honor points
✅   for : 
- Complete reset of arena and honor points at the beginning of each season.
**AND**
- Previous Arena gear purchasable with honor points, **supplemented by a small allocation of arena points**, to incentivize participation in the arena without **nullifying the considerable efforts of those who several months of the previous season to obtain this gear**.
Basic example can be: 
One arena item ==>  **50hk honor point + 400 arena point**

**Reply by TotallyNotMyAccount** [reply]
I'm liking this one. 350-450 pts for an item, depending on slots sounds about right. Maybe up to 600 for a 2h. Honor cost could be cut down to 20-30k imo. But really, I agree with the general thrust of this.

**Reply by balleny** [reply]
600 seems a bit low to me, same with 350 points. it should be slightly above doing the bare minimum per week to get more than 1 item per week
(hence 6 weeks for the full set given that you need like 1 initial/additional week) - and weapons in a similar manner, all 3 slots in total should be about 6 weeks doing bare minimum effort

**Reply by TotallyNotMyAccount** [reply]
That's one item per week, if you are making minimum wage (250ish from arena + 170 pts from CTA and daylie win). Which sounds reasonable.

**Reply by balleny** [reply]
ye, i am usually taking the min points of 5s as a baseline, hence the higher value

**Comment by DPComputer** [comment]
Hope you do realize that taking time to try and customize your PVP experience would take away time/workload from what the whole scope of CC is: fixing and releasing the 3.3.5 core.

I do understand the frustration of not enjoying your PVP experience on CC, but this is a WoTLK server first and foremost, very focused on the PVE experience of it (Raids, Dungeons, etc).

And before you even think of gatekeeping my comment with something along the lines of "another PVE player who doesn't even play PVP who thinks he knows better!" let me use an analogy:

I'm not a military pilot either, but if i see a military jet zooming throught skyscrapers on a city i can definately tell it's not a good idea.

**Reply by tbcxd** [reply]
These changes do not take very much of the devs time.
We didn't ask for any custom pvp events, soloq or other time consuming changes.
The changes that we request are pretty easy to provide and could potentially increase the population of CC which we all want, right?
I am mostly a pvp guy, but I always attend to raids and so do the 90% of the pvp community.

And btw, the admins themselves requested us to make suggestions here, so if they thought it would be too time consuming, they wouldn't have done it in the first place.

**Reply by balleny** [reply]
> These changes do not take very much of the devs time.

then take the commit from s1, exchange all ids and values and provide it ready tailored if it aint taking much time anyway

**Reply by tbcxd** [reply]
You know what I meant, don't get the discussion in that way.
Changing some values compared to scripting a whole event from 0 or scripting soloq, custom arenas and stuff, the first takes a lot less time.
I would 100% do it, if I had more knowledge in this particular area, but sadly, I do not.

I have stated multiple times that I really appreciate the devs work and the whole idea behind the CC server.

**Comment by balleny** [comment]
mostly duplicate content
https://github.com/chromiecraft/chromiecraft/discussions/6655#discussioncomment-8435514

**Comment by Amrx2** [comment]
In between seasons, S1 gear is being discounted.

**Reply by balleny** [reply]
always good to specify the timeframe because there was no real communication about a timeline related to pvp yet
when season ends, when items get introduced, when discounts apply, when season restarts, when last AP flush takes place
